zerver/lib/events.py

```diff
@@ -149,8 +149,8 @@ def fetch_initial_state_data(user_profile, event_types, queue_id,
 
     return state
 
-def apply_events(state, events, user_profile):
-    # type: (Dict[str, Any], Iterable[Dict[str, Any]], UserProfile) -> None
+def apply_events(state, events, user_profile, include_subscribers=True):
+    # type: (Dict[str, Any], Iterable[Dict[str, Any]], UserProfile, bool) -> None
     for event in events:
```

Comment:
yeah, I think we'll probably want to do that refactoring, since it'd save us a layer of giant refactoring.